propka/input.py

```diff
@@ -67,14 +72,17 @@ def read_molecule_file(filename: str, mol_container, stream=None):
         usually have an associated file name, an appropirate file name should
         be passed to the ``filename`` argument. In this case, ``filename`` is
         not opened for reading, but instead is used to help recognise the file
-        type (based on the extension being either `.pdb` or `.propka_input`)
-        and also uses that given ``filename`` to assign a name to the input
+        type (based on the extension being `.pdb`) and also uses that given
+        ``filename`` to assign a name to the input
         :class:`~propka.molecular_container.MolecularContainer` object.
 
         >>> read_molecule_file('test.pdb', mol_container,
                                stream=string_io_object)
         <propka.molecular_container.MolecularContainer at 0x7f6e0c8f2310>
 
+
+    .. versionchanged:: 3.4.0
+       PROPKA input files (extension: `.propka_input`) are no longer read.
     """
     input_path = Path(filename)
     mol_container.name = input_path.stem
```

Comment:
Testing inputs by extension is probably not really necessary anymore. Thoughts about just removing this logic and just assuming everything being passed in a PDB file?

(that being said, removing the need for `filename` is going to break the API again...)

Reply:
I think we should just leave it as-is for now.